test/fixtures/filterings.yml

```diff
@@ -373,3 +373,11 @@ crud_hosts_1_2:
 crud_hosts_1_3:
   filter: crud_hosts_1
   permission: view_hosts
+<% Foreman::Plugin.all.map(&:default_roles).inject({}, :merge).each do |role,permissions| %>
```

Comment:
I had no idea this could be done in the yml files - great job. I thought about adding these fields through a script in the helper or something but this is definitely easier